<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdministracionBodega</class>
 <widget class="QWidget" name="AdministracionBodega">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="EliminarPushButton">
   <property name="geometry">
    <rect>
     <x>473</x>
     <y>76</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar </string>
   </property>
  </widget>
  <widget class="QPushButton" name="ActualizarPushButton">
   <property name="geometry">
    <rect>
     <x>473</x>
     <y>43</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NuevoPushButton">
   <property name="geometry">
    <rect>
     <x>473</x>
     <y>10</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="BodegasTreeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>457</width>
     <height>318</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="rootIsDecorated">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">5</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
